internal/generator/ecdsa/template/ecdsa.go.tmpl

```diff
@@ -256,16 +261,21 @@ func (publicKey *PublicKey) Verify(sigBin, message []byte, hFunc hash.Hash) (boo
 
 	sInv := new(big.Int).ModInverse(s, order)
 
-	// compute the hash of the message as an integer
-	dataToHash := make([]byte, len(message))
-	copy(dataToHash[:], message[:])
-	hFunc.Reset()
-	_, err := hFunc.Write(dataToHash[:])
-	if err != nil {
-		return false, err
-	}
-	hramBin := hFunc.Sum(nil)
-	m := HashToInt(hramBin)
+	m := new(big.Int)
```

Comment:
It is sufficient here to do `var m *big.Int` and not allocate. You assign value to m later. This allocation will be wasted.